mtw2-translator.py

```diff
@@ -26,11 +33,14 @@ def translate_file(filename, newfilename, source_lang, target_lang, ref_translat
                         text = value
                         keys_intact = keys_intact + 1
                     else:
+                        comment = f"¬>>>>> AUTO TRANSLATION >>>>> {value}\n"
```

Comment:
In my opinion only this change is useful. Other features are not necessary at this point.